vllm/attention/ops/triton_flash_attention.py

```diff
@@ -139,7 +139,7 @@ def _attn_fwd_inner(
                 size_n = start_n + OFFS_N[None, :]
                 mask = size_n < boundary_m[:, None]
                 qk = tl.where(mask, qk, float("-inf"))
-        if IS_CAUSAL:
+        if IS_CAUSAL:  # noqa: SIM102
```

Comment:
Is this required?

Reply:
nope.

Reply:
Let's remove the `noqa` then.

Reply:
I think this is already addressed at latest change.

Reply:
You're right, never mind then.